MediaBrowser.Model/Session/GeneralCommand.cs

```diff
@@ -14,14 +14,16 @@ public GeneralCommand()
     }
 
     [JsonConstructor]
-    public GeneralCommand(Dictionary<string, string> arguments)
+    public GeneralCommand(Dictionary<string, string>? arguments = null)
     {
-        Arguments = arguments;
+        if (arguments != null) {
+            Arguments = arguments;
+        }
     }
 
     public GeneralCommandType Name { get; set; }
 
     public Guid ControllingUserId { get; set; }
 
-    public Dictionary<string, string> Arguments { get; }
+    public Dictionary<string, string> Arguments { get; } = new();
```

Comment:
Better to do it in the constructor.

```suggestion
    public Dictionary<string, string> Arguments { get; }
```